pkg/bridge/bridge.go

```diff
@@ -6,19 +6,46 @@ import (
 
 	"github.com/vishvananda/netlink"
 	"k8s.io/klog"
+
+	"github.com/rancher/harvester-network-controller/pkg/dhcp"
 )
 
 type Bridge struct {
 	*netlink.Bridge
+	DHCPSignal chan bool
 }
 
 func NewBridge(name string) *Bridge {
 	vlanFiltering := true
-	return &Bridge{
+	br := &Bridge{
 		Bridge: &netlink.Bridge{
 			LinkAttrs:     netlink.LinkAttrs{Name: name},
 			VlanFiltering: &vlanFiltering,
 		},
+		DHCPSignal: make(chan bool),
+	}
+
+	go br.StartDHCPDaemon()
+
+	return br
+}
+
+// StartDHCPDaemon maintain dynamic ip address of bridge with dhcp
+func (br *Bridge) StartDHCPDaemon() {
```

Comment:
prefer to call it `StartDHCPDaemonClient`

Reply:
may be `StartDHCPClientDaemon` is better